docs/guides/src/main/server/logging.adoc

```diff
@@ -114,51 +128,55 @@ By default, the console log handler logs plain unstructured data to the console.
+Colored console log output for unstructured logs is disabled by default. Colors may improve readability, but they can cause problems when shipping logs to external log aggregation systems. If you want to enable or disable color-coded console log output, enter following command:
 
 <@kc.start parameters="--log-console-color=<false|true>"/>
 
 == File logging
-Instead of logging to the console, Keycloak also supports unstructured logging to a file.
+As an alternative to logging to the console, you can use unstructured logging to a file.
 
 === Enable file logging
-Logging to a file is disabled by default. To enable it, run the following command:
+Logging to a file is disabled by default. To enable it, enter the following command:
 
 <@kc.start parameters="--log=console,file"/>
 
-=== Configuring path and name of the generated log file
-By enabling the file log handler, a log file named `keycloak.log` will be created inside the `data/log` directory of your Keycloak installation.
+A log file named `keycloak.log` is created inside the `data/log` directory of your Keycloak installation.
 
-To change the location and name of the generated log file, run the following command:
+=== Configuring the location and name of the log file
 
-<@kc.start parameters="--log=console,file --log-file=<path-to>/<your-file.log>"/>
+To change where the log file is created and the file name, perform these steps:
 
-Please make sure the location for the logfile is writeable. If not, an error will be thrown at start-up. Keycloak will start correctly, but no file containing logs will be created.
+. Find a writable directory to store the log file.
++
+If the directory is not writable,  Keycloak will start correctly, but no log file is created.
+
+. Enter this command:
++
+<@kc.start parameters="--log=console,file --log-file=<path-to>/<your-file.log>"/>
 
 === Configuring the file handler format
-You can configure a different logging format for the file log handler by running the following command:
+To configure a different logging format for the file log handler, enter the following command:
 
 <@kc.start parameters="--log-file-format=<pattern>"/>
 
-Please see the <<Configuring the console log format>> section in this guide for more information and a table of the available pattern configuration.
+For details on the available pattern configuration, see <<Configuring the console log format>>.
 
-== Configuring raw quarkus logging properties
-At the time of writing, the logging features of the quarkus based Keycloak are basic, yet powerful. Nevertheless, expect more to come and feel free to join the https://github.com/keycloak/keycloak/discussions/8870[discussion] at GitHub.
+== Raw quarkus logging properties
```

Comment:
Pls remove this whole section. we are including json logging and file logging now, and i don't think we should advertise using raw properties anymore but instead let them create issues on our side.